java/src/jmri/jmrit/logixng/actions/configurexml/ActionMemoryXml.java

```diff
@@ -102,6 +76,8 @@ public boolean load(Element shared, Element perNode) throws JmriConfigureXmlExce
         String uname = getUserName(shared);
         ActionMemory h = new ActionMemory(sys, uname);
 
+        LogixNG_SelectTableXml selectTableXml = new LogixNG_SelectTableXml();
```

Comment:
Not a request for a change, but I've started using `var` for local variables like this.  It saves having to insert another import. Particularly helpful with more complicated definitions, like generics.

Reply:
Thanks for the tip!